Makefile

```diff
@@ -279,6 +284,13 @@ manifest:
 	$(CURDIR)/hack/generate-manifest-octant.sh --mode dev > build/yamls/antrea-octant.yml
 	$(CURDIR)/hack/generate-manifest-windows.sh --mode dev > build/yamls/antrea-windows.yml
 
+.PHONY: scale-manifest
+scale-manifest:
+	@echo "===> Generating dev manifest for Antrea <==="
+	$(CURDIR)/hack/generate-manifest.sh --mode dev --simulator > build/yamls/antrea-simulator.yml
+	$(CURDIR)/hack/generate-manifest.sh --mode dev --ipsec > build/yamls/antrea-ipsec.yml
+	cp $(CURDIR)/build/yamls/patches/simulator/antrea-agent-simulator.yml build/yamls/
```

Comment:
why don't merge the simulator yaml into "antrea-simulator.yml" when executing generate-manifest.sh? I think it's how other modes do.

Reply:
since simulator is not in antrea.yml, patch will fail due to it can not find simulator.

Reply:
@tnqn how about command like:

cat $(CURDIR)/build/yamls/patches/simulator/antrea-agent-simulator.yml >> build/yamls/antrea-simulator.yml

this can also merge antrea-agent-simulator.yml into antrea-simulator.yml